.gitignore

```diff
@@ -6,4 +6,5 @@ coverage/
 types/
 test/runtime.js
 packages/babel-plugin-jsx-dom-expressions/index.js
-!packages/babel-plugin-jsx-dom-expressions/types/
\ No newline at end of file
+!packages/babel-plugin-jsx-dom-expressions/types/
+package-lock.json
```

Comment:
This kept popping up, figured I'd ignore them since they aren't checked in (as it should be with libs)